addons/stock/models/stock_move.py

```diff
@@ -2161,3 +2161,14 @@ def _get_moves_orig(self, moves=False):
         for move in self.move_orig_ids:
             moves |= move._get_moves_orig(moves)
         return moves
+
+    def _get_aggregated_product_properties(self):
+        self.ensure_one()
```

Comment:
We need to leave the method exactly as it was:
- Receive both move and move line
- Don't change inside code
- Leave it into the `stock.move.line` model

The above is because this is targeting a stable version, so we need to  follow the "just minimal changes" policy

Reply:
@luisg123v I already made the changes